### PR TITLE
Remove SSLv3 usage

### PR DESCRIPTION
**Pull request by imsickofmaps**
Test and migrate to TLS to Jembi

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -22,11 +22,11 @@
...
     "vumigo_v02": "~0.2",
-    "q": "~0.9",
+    "q": "~0.9.7",
     "lodash": "~2.4.1",
-    "moment": "~2.6.0",
-    "go-jsbox-metrics-helper": "~0.1.1",
-    "libxmljs": "0.8.x"
+    "moment": "~2.6",
+    "go-jsbox-metrics-helper": "^0.1.0",
+    "libxmljs": "~0.8.1"
```
Comment (hodgestar) [inline package.json:29]: What's with all the version changes, and more specifically, why did `go-jsbox-metrics-helper` go back to `0.1.0`?

**Comment by hodgestar** [comment]
Other than the question 'bout the version changes, looks good.

File: package.json (hunk 1 above)
Comment (imsickofmaps) [inline package.json:29]: I wanted to check that we weren't tricking ourselves with something that wasn't in prod and so I checked the version in the base of praekelt/vumi-go and that's what it is

Comment (hodgestar) [inline package.json:29]: Ah, these were just copied across from Vumi Go's package.json to be safe. I think that's okay.

**Comment by hodgestar** [comment]
:+1:

**Pull request merged**